### PR TITLE
Performance optimization

### PR DESCRIPTION
**Pull request by benzon**
Added canSleep vvar to DrawMarker section, lowers performance usage, added a "canSleep" to enter/exit marker, tapping in to HasAlreadyEnteredMarker.

When out of DrawDistance 0.01 ms usage, when in DrawDistance 0.03ms

Insted of 0.05 ms all the time.

Changed DrawDistance to 50, no reason to set it to 100 bad practice

### CODE REVIEW

**Comment by benzon** [comment]
@ElPumpo Performance improvement PR actually thing this should be done in most script that uses DrawMarker, since it will give a improvement in general, the same goes for the EnteredMarker.

And lower DrawDistance in general, it's a bad idea with 100.

File: client.lua [hunk 1]
```diff
@@ -89,6 +95,10 @@ Citizen.CreateThread(function()
 
 			TriggerEvent('esx_teleportpads:hasExitedMarker', LastPad, LastAction)
 		end
+
+		if not HasAlreadyEnteredMarker then
+            Citizen.Wait(500)
```
Comment (HawaiiBeach) [inline client.lua:100]: Uuuum fix this indent

Comment (benzon) [reply client.lua:100]: let me fix that not sure why the heck it changed that

**Comment by benzon** [comment]
@ElPumpo there you go, added .editorconfig to, looks like the indent was messed up from the start.

**Comment by ElPumpo** [comment]
Lol you just ruined this PR. Sorry dude I wont merge this re make your pr and send it back, but thanks for your efforts!!

**Comment by benzon** [comment]
Okay how the heck did i ruin it? by fixing the ident to use the same style as esx ?

**Comment by ElPumpo** [comment]
You didnt fix it at all. You switched to 4 space

**Pull request closed without merging**